torch/_tensor.py

```diff
@@ -848,25 +856,42 @@ def unique_consecutive(self, return_inverse=False, return_counts=False, dim=None
         )
 
     @_handle_torch_function_and_wrap_type_error_to_not_implemented
-    def __rsub__(self, other):
+    def __rsub__(self, other: Any) -> "Tensor":
         return _C._VariableFunctions.rsub(self, other)
 
     @_handle_torch_function_and_wrap_type_error_to_not_implemented
-    def __rdiv__(self, other):
+    def __rdiv__(self, other: Any) -> "Tensor":
         return self.reciprocal() * other
 
     __rtruediv__ = __rdiv__
     __itruediv__ = _C._TensorBase.__idiv__
 
-    __pow__ = _handle_torch_function_and_wrap_type_error_to_not_implemented(
-        _C._TensorBase.pow
+    # TODO Currently there's no standard solution to combine ParamSpec and overload.
+    # mypy will only pass the first overload to ParamSpec, making `__pow__` and
+    # `__ipow__` only accept `exponent: Tensor`. Therefore, we explicitly annotate
+    # them and force the linter to accept so that end-users can see it correct.
+    # However, we directly use Any here, otherwise there'll be too many to suppress.
+
+    def __pow__(self, exponent: Any) -> "Tensor":
+        ...
+
+    __pow__ = (  # noqa: F811
+        _handle_torch_function_and_wrap_type_error_to_not_implemented(
+            _C._TensorBase.pow
+        )
     )
-    __ipow__ = _handle_torch_function_and_wrap_type_error_to_not_implemented(
-        _C._TensorBase.pow_
+
+    def __ipow__(self: Self, exponent: Any) -> Self:
+        ...
+
+    __ipow__ = (  # noqa: F811
+        _handle_torch_function_and_wrap_type_error_to_not_implemented(  # pass UFMT
```

Comment:
Why `# pass UFMT` is needed here but on on line 879)?
```suggestion
        _handle_torch_function_and_wrap_type_error_to_not_implemented(
```

Reply:
Otherwise, UFMT requires `# noqa: F811` moved onto line 888. However, FLAKE8 and RUFF need this noqa on Line 887.
So it's needed to pass UFMT

Reply:
Another option is to change `_handle_torch_function_and_wrap_type_error_to_not_implemented` to a shorter name. Any suggestions?

Reply:
> Why `# pass UFMT` is needed here but on on line 879)?

Sorry, seems I didn't positively answer this question. They are different in that line 889 has a `# type: ignore[arg-type]`, but not on 880. Therefore this is added to 888 but not 879.

However, I tried a little more and found out that we can remove the `type: ignore` after #103376 is landed.